Plandomizer.py

```diff
@@ -408,19 +408,32 @@ def alter_pool(self, world, pool):
         else:
             self.pool_add_item(pool, '#Bottle', -bottles)
 
+        bottles = 0
         for item_name, record in self.starting_items.items():
             if bottle_matcher(item_name):
-                self.pool_remove_item([pool], "#Bottle", record.count)
+                if (world.zora_fountain != "open"
+                        and world.item_pool_value == "plentiful"
+                        and "Bottle with Letter" not in self.starting_items
+                        and record.count > 2):
+                    record.count -= 1
+                bottles = record.count
             elif trade_matcher(item_name):
                 self.pool_remove_item([pool], "#AdultTrade", record.count)
             elif IsItem(item_name):
                 try:
-                    self.pool_remove_item([pool], item_name, record.count)
+                    if item_name == "Bottle with Letter":
+                        self.pool_remove_item([pool], item_name, 2)
```

Comment:
Removing 2 Letters from the pool... is that guaranteed to work? And does it matter how many Bottles with Letter the player asked to start with?